charts/lock-unlock-rewrite/values.yaml

```diff
@@ -74,11 +74,13 @@ ingress:
 
 resources:
   limits:
-    memory: 2Gi
+    memory: 16Gi
   requests:
     cpu: 50m
     memory: 2Gi
 
+javaOptions: "-Xmx16g -Xms4g"
```

Comment:
Wil je deze nog onder `image` plaatsen? 't Is geen setting van de deployment zelf, maar van de container

Reply:
Hmm, `resources` is iets voor Kubernetes en gaat niet over de inhoud van de `image`. Ik moet bekennen, er zijn hier niet echt regels voor wat waar thuis hoort, bij sensrnet hadden we bijvoorbeeld `settings:` waaronder we de container settings plaatsten, dat was ook wel mooi. 

Heb de Helm best practices bekeken (https://helm.sh/docs/chart_best_practices/values/#flat-or-nested-values), maar daar prefereren ze ook flat boven nested. Dus in dat geval is dit helemaal prima 🙂

Wat mij betreft prima om 'm zo te laten staan en alleen de Chart version te updaten!

Reply:
Scherp om even naar de best practices te kijken. Daar had ik niet aan gedacht. Top! Ga ik 't afmaken met een release(je) 😃